src/DynamoCore/Models/AnnotationModel.cs

```diff
@@ -15,10 +16,13 @@ namespace Dynamo.Models
     public class AnnotationModel : ModelBase
     {
         #region Properties
+        public event Func<Guid,ModelBase> GetModelBaseEvent;      
         public double InitialTop { get; set; } //required to calculate the TOP position in a group         
-        public double InitialHeight { get; set; } //required to calculate the HEIGHT of a group 
-        private string modelGuids { get; set; }   
+        public double InitialHeight { get; set; } //required to calculate the HEIGHT of a group          
         private const double doubleValue = 0.0;
+        private const double minHeight = 20.0;
+        private const double extendSize = 10.0;
+        public List<ModelBase> DeletedModelBases { get; set; }
```

Comment:
`DeletedModelBases` needs, at the very least, some documentation on why it is there.  It's really difficult for me to tell.